tests/test_coupon.py

```diff
@@ -199,7 +199,7 @@ def setUp(self):
         if not os.path.exists(filename):
             # Change this path to the serial port and set the baudrate used by
             # fiscal printer when recreating the tests.
-            real_port = SerialPort('/tmp/stoq-ecf')
+            real_port = SerialPort('/dev/ttyUSB0')
```

Comment:
tests must go to a file, Why are you pointing to a /dev/^*?
Why all files for tests changed? can you explain a little bit more extended in the body of the PR please?